### DISCUSSION TITLE
Scheil calculation?

### DISCUSSION THREAD

**Discussion by MWW726** [pycalphad/pycalphad#387] [Q&A] [answered]
Hi,
    I have a question. can pycalphad do a Scheil calculation using a simple way? I have not seen the example of Scheil.

**Comment by bocklund** [comment, marked as answer]
We have an additional package called `scheil` that can do multi-component Scheil calculations. You can find the documentation site here: https://scheil.readthedocs.io/en/latest/ and source code here: https://github.com/pycalphad/scheil

**Reply by MWW726** [reply]
OK, it looks great. But, why not put all big function on the pycalphad official website because many people are accustomed to reading official website and example of the website. =-=

**Reply by bocklund** [reply]
Thank you for the feedback. I'm hesitant to add a full example to the main pycalphad documentation because `scheil` is a separate install, but I think it would be a good idea to mention and add a link to it in the pycalphad documentation. There are actually a couple of officially-supported and community-supported tools that use pycalphad that would be nice to add to the pycalphad website.

@richardotis any comments?
